14/mysql-tomcat/Dockerfile

```diff
@@ -103,6 +103,7 @@ VOLUME /usr/local/xwiki
 #            configure xwiki's hibernate.cfg.xml file.
 # - CONTEXT_PATH: The name of the context path under which XWiki will be deployed in Tomcat. If not specified then it'll
 #                 be deployed as ROOT.
+# - ADDITIONAL_JDBC_PARAM: Additional JDBC parameters. An example could be '&allowPublicKeyRetrieval=true'
```

Comment:
I would not add the `&`prefix. Instead it would be nicer to add it automatically in the code below if `ADDITIONAL_JDBC_PARAM` is not empty. WDYT?